src/main/java/com/example/gradecalculator/entities/SchoolYear.java

```diff
@@ -26,7 +26,7 @@ public class SchoolYear {
     @OneToMany(mappedBy = "schoolYear")
     private List<Subject> subjects;
 
-    public SchoolYear() {
+    public SchoolYear(long l, String s, GradeSystem gradeSystem) {
```

Comment:
also a constructor that has parameters but does never use those parameters

Reply:
used in test class

Reply:
but values are never assigned. use empty constructor instead

Reply:
values are assigned

Reply:
where? it even says in IntelliJ: 'Parameter is never used'